packages/ninjamsrv/PKGBUILD

```diff
@@ -0,0 +1,42 @@
+# Maintainer: OSAMC <https://github.com/osam-cologne/archlinux-proaudio>
+# Contributor: jakob <grandchild@gmx.net>
+# Contributor: Florian Hülsmann <fh@cbix.de>
+
+_name=ninjam
+pkgname=ninjamsrv
+pkgver=0.080
+pkgrel=1
+pkgdesc='Ninjam online jamming server'
+arch=(x86_64 aarch64)
+url='https://cockos.com/ninjam/'
+license=(GPL2)
+depends=(gcc-libs)
```

Comment:
Waiting for more feedback from IRC archlinux-aur on whether having `systemd` in `depends` (probably not) or `optdepends` is useful...

Reply:
`depends` doesn't seem to make sense from [it's definition](https://wiki.archlinux.org/title/PKGBUILD#depends). You can run `ninjamsrv` without systemd. Then, `optdepends` IMO doesn't make much sense either because every Arch installation has systemd (see `pacman -Si base`). There are cases where systemd is not installed (non-systemd derivatives, archlinux docker container) but in all of these it's not desired to use systemd for running ninjamsrv.

Reply:
Ok, agreed, that systemd is a special case. Though a package being in `base` apparently doesn't preclude it from being in `depends. See `gcc-libs`.

Reply:
TBH even the gcc-libs dependency really only makes sense if it's properly versioned or uses soname depends, so whenever the required gcc-libs version is not there an install would fail 🤔 But that would require the gcc-libs package to properly list all its libprovides :D